ONSdigital/eq-author-app
# Remove perm scrollbars & scrollbar styling

### What is the context of this PR?

PR #1402  needs to be merged first - also fixes the build error

Removes the custom styling for the Author scroll bars

I've also done `EAR-856` in this ticket to save opening another pull request. 

The following places had permanent scrollbars:

**SectionMenu**
`/eq-author/src/components/ContentPickerv2/DestinationPicker/SectionMenu.js`
**SectionMenu**
`/eq-author/src/components/ContentPickerv2/SectionMenu.js`
**SharePage**
`/eq-author/src/App/sharing/SharePage.js`
**QcodesTable**
`/eq-author/src/App/qcodes/QcodesPage.js`
**MetadataPage**
`/eq-author/src/App/metadata/MetadataPage.js`

### How to review 
1. Create a questionnaire
2. Check scrollbars have native OS/Browser styling
3. Check for the permanent scrollbars above
